.env

```diff
@@ -38,7 +38,9 @@ ASSIGNMENT_STATE_INTERVAL_THRESHOLD=P1D
 ###> LTI ###
 LTI_ENABLE_INSTANCES_LOAD_BALANCER=false
 LTI_LAUNCH_PRESENTATION_LOCALE=en-EN
-LTI_LAUNCH_PRESENTATION_RETURN_URL='http://localhost:8000/index.html'
+LTI_LAUNCH_PRESENTATION_RETURN_URL=http://localhost:8000/index.html
+# Possible values for LTI Instance load balancer strategy: username or userGroupId
```

Comment:
`username` and `usergroup` looks more nicely

Reply:
Also, does it make sense to provide `.env.example` instead of changing the real .env file?

Reply:
About usergroup: I like the idea, however it's not camelCase. username is a word on it's own, but usergroup is invalid, userGroup would be the appropriate format :(
About the .env file: It contains default, but working implementation, but I agree that it could be improved. I'll create a ticket for that 👍